File: Java/Fase2.java

import java.util.*;

public class Fase2 {
	public static void main(String[] args) {
		char[] name = { 'f','a','r','l','e','y', '0' };

		List<Character> listName = new ArrayList<>();

		for (int i = 0; i < name.length; i++) {
			listName.add(name[i]);
		}

		Iterator<Character> it = listName.iterator();

		while (it.hasNext()) {
			Object aux = it.next();
			if ("AEIOUaeiou".contains(aux.toString())) {
				System.out.println("Vocal");
			} else if ("0123456789".contains(aux.toString())) {
				System.out.println("Els noms de persones no contenen números!");
			} else {
				System.out.println("Consonante");
			}
		}
	}
}

    

File: JavaScript/Fase2.js

function funcionFase2(){
			
	const name = ['f','a','r','l','e', 'y','0'];
	const vocal = 'aeiouAEIOU';
	const num = '0123456789';

	for (var i = 0; i < name.length; i++) {
		if(vocal.indexOf(name[i]) != -1){
		console.log("Vocal");
		} else if(num.indexOf(name[i]) != -1){
		console.log("Els noms de persones no contenen números!");
		} else{
		console.log("Consonante");
		}
	}
}



    
